files/ui/advancedpage.ui

```diff
@@ -454,6 +454,16 @@
                 </property>
                </widget>
               </item>
+              <item row="3" column="1">
+               <widget class="QCheckBox" name="antialiasAlphaTestCheckBox">
+                <property name="toolTip">
+                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enables alpha testing for smoother anti-aliasing (Requires anti-aliasing to be enabled)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
This description is bogus. It does antialiasing on alpha tested things instead of having alpha tested things be aliased and ugly.

Reply:
No longer an issue?

Reply:
I updated the description based off what I saw [here](https://openmw.readthedocs.io/en/latest/reference/modding/settings/shaders.html#antialias-alpha-test).
Does this look better @AnyOldName3?